CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,224 @@
+
+**Requirements**
+- Member for at least **3 months**.
+- Regular, high-quality reviews in the specialty.
+- several meaningful PR reviews over the last 3 months (or equivalent specialty output).
+
+**Privileges**
+- Listed as `reviewer` in the relevant `OWNERS` files.
+- May use `/lgtm` on PRs within the specialty.
+- Eligible for **Maintainer** nomination.
+
+> A contributor may hold different roles across specialties (e.g., **Reviewer** in CI, **Member** in Core Controllers).
+
+---
+
+### 4) Maintainer (Project-Wide)
+Project leaders with governance, release, and cross-specialty responsibility.
+
+**Requirements**
+- Reviewer for at least **6 months** in one or more specialties.
+- Demonstrated leadership, reliability, and constructive collaboration.
+- Nominated and approved by a **supermajority** of Maintainers.
+
+**Privileges**
+- GitHub admin rights as needed.
+- Release management authority.
+- OSC / FOSSA Administrator
+- Representation within CNCF.
+
+---
+
+## Specialty Tracks
+
+Specialties define scope for `reviewer` permissions and expectations. 
+
+### CI / Infrastructure
+Focus: GitHub Actions, build/test pipelines, images, release automation.
+Activity: Reviews CI changes, enforces reproducibility, flags flaky tests.
+
+### Testing
+Focus: Unit/integration/E2E tests, frameworks, fixtures, test data.
+Activity: Ensures adequate coverage and quality in PRs, promotes testability.
+
+### Core Controllers
+Focus: CRDs, reconciliation logic, API evolution, performance.
+Activity: Reviews controller/CRD changes, ensures API consistency and backward compatibility.
+
+### Providers
+Focus: Provider integrations (AWS, Vault, GCP, Azure, CyberArk, etc.).
+Activity: Reviews provider-specific code and conformance to provider guidelines; coordinates breaking changes (for providers that aren't `stable` graded).
+
+---
+
+## Interim Roles
+
+In some cases, Maintainers may create **interim roles** for **Members**, **Reviewers** in a given specialty or **Maintainers**.  
+These are **temporary training-oriented roles** designed to help contributors gain the experience needed to meet the full role requirements.
+
+### Purpose
+- Provide **on-the-job training** and mentorship.
+- Allow contributors to participate with limited privileges while building their track record.
+- Reduce barriers for new contributors to join governance roles.
+
+### Scope
+- Limited to a **maximum of three (3) months** for **Members** and **Reviewers**, and **a maximum of six (6) months** for **Maintainers**.
+- Specialty and scope are explicitly documented in `OWNERS` files and/or a public tracking issue.
+- Interim roles per specialty can accumulate (e.g. a contributor can be an interim reviewer on CI and on Providers at the same time). This is to allow a fast path to upskill future project-wide maintainers.
+- Interim roles are not eligible for advancement to permanent roles (an interim reviewer cannot be eligible to be a maintainer).
```

Comment:
I'm not sure I follow the wording here. You are saying they cannot advance to permanent roles. But then in the parenthesis you say they cannot jump roles?

Reply:
what I meant is:

Interim Member is not elegible for a permanent reviewer role;
Interim reviewer is not elegible for a permanent maintainer role;

i.e. we cannot abuse the interim system to get permanent roles easily by promoting them